dev/environment-base.yaml

```diff
@@ -0,0 +1,12 @@
+name: bio-core-dev
+channels:
+  - pytorch
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
Should we add pytorch in the dependencies? Or remove the pytorch channel?

Reply:
right, in principle we should test

* no NN backend
* Pytorch
* onnxruntime
* tensorflow2
?